### ISSUE TITLE
Can I share like "Boost" in Mastodon(or "repeat" in GNU social)?

### ISSUE THREAD

**Issue by roytam1** [friendica/friendica#5873]
I'm using squeet.me friendica instance and when I use "share" button, Mastodon contacts can't receive a new post, or being notified with boosted/repeated of original post.

**Comment by MrPetovan** [comment]
You're supposed to be able to, thanks for the report!

**Comment by annando** [comment]
The "Announce" type in ActivityPub is currently not supported by Friendica. The background is a fundamental difference in how Friendica/Diaspora and ActivityPub are handling the repeating of messages.

**Comment by hoergen** [comment]
Is there a way how you can achieve that?

@MrPetovan I don't think this is a bug. It is like a reshare feature, what we have actually (like on twitter) but the repeat function is a new function to friendica, so it would be better to put here "new feature"?

**Comment by MrPetovan** [comment]
Good point, although I think you could have switched labels yourself without having to wait on me.

**Comment by hoergen** [comment]
You are right but I wanted to wait you point of view. Because there could have been other reasons, that I didn't see. Thanks for changing it  🌻

**Comment by MrPetovan** [comment]
You were completely right, in my first comment I confused quoting a post (that we do with "Share") and forwarding a post (that people do with "Boost", "Repeat").

**Comment by annando** [comment]
In fact we now are doing an "Announce" message when we repeat a message. But since our repeats are creating new posts with a new url, while the AP "announces" are simply resharing the existing url, we currently are creating two posts out of it - while the second one will appear at AP sites, whenever someone from Friendica comments.

And yeah, this is complicated and not understandable - and ugly.

**Comment by annando** [comment]
I think the original issue is mostly solved. Until we completely rebuild the resharing mechanism there is no way to improve it further.
